linkerd/app/inbound/src/direct.rs

```diff
@@ -261,7 +267,7 @@ impl std::error::Error for RefusedNoIdentity {}
 
 impl std::fmt::Display for RefusedNoTarget {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "A named target must be provided on gateway connections")
+        write!(f, "T named target must be provided on gateway connections")
```

Comment:
oops
```suggestion
        write!(f, "A named target must be provided on gateway connections")
```

Reply:
hahaha, i should probably not have used find-and-replace for this!